src/main/kotlin/com/criptext/mail/scenes/emaildetail/workers/DownloadAttachmentWorker.kt

```diff
@@ -37,7 +37,7 @@ class DownloadAttachmentWorker(private val fileSize: Long,
                                private val downloadPath: String,
                                private val accountDao: AccountDao,
                                private val storage: KeyValueStorage,
-                               private val httpClient: HttpClient,
+                               httpClient: HttpClient,
```

Comment:
why is not private now?

Reply:
it's not necessary, since the http client is only used on initialization of the worker, in this cases kotlin suggests that private val is not needed.